api/src/handlers/server_api.rs

```diff
@@ -62,17 +62,20 @@ impl Handler for KernelDownloadHandler {
 /// Status handler. Post a summary of the server status
 /// GET /v1/status
 pub struct StatusHandler {
-	pub chain: Weak<chain::Chain>,
+	pub chain: Weak<Chain>,
 	pub peers: Weak<p2p::Peers>,
-	pub sync_state: Weak<chain::SyncState>,
+	pub sync_state: Weak<SyncState>,
 }
 
 impl StatusHandler {
 	fn get_status(&self) -> Result<Status, Error> {
 		let head = w(&self.chain)?
 			.head()
 			.map_err(|e| ErrorKind::Internal(format!("can't get head: {}", e)))?;
-		let sync_status = w(&self.sync_state)?.status();
+		let sync_status = match w(&self.sync_state)?.status() {
+			SyncStatus::NoSync => "no_sync".to_owned(),
```

Comment:
Might be worth getting some feedback around `no_sync` to see if this makes sense to people.
Maybe it should just be `running` or something as this is going to be the most common state people see (presumably).

Reply:
Yeah agree, to be fair the last commit was just that I wanted to save my work before testing the new master :). Will ask for feedback.